doc/tutorials/inter-vm_communication.rst

```diff
@@ -0,0 +1,486 @@
+.. _inter-vm_communication:
+
+ACRN Inter-VM Communication
+##############################################
+
+ACRN supports three kinds of Inter-VM communication – Inter-VM vUART,
+Inter-VM network communication, and Inter-VM shared memory device (ivshmem).
+Each communication has its advantage and shortage and is listed below:
+
+Inter-VM vUART
+**************
+
+Inter-VM vUART communication is based on the vUART implementation. It is
+used to transfer data between two VMs at low speed. (Refer to :ref:`vuart_virtualization`
+and :ref:`vuart_config`).
+
+:numref:`Inter-VM vUART communication` shows the Inter-VM vUART communication overview:
+
+.. figure:: images/Inter-VM_vUART_communocation_overview.png
+   :align: center
+   :name: Inter-VM vUART communication
+
+   Inter-VM vUART communication
+
+- Pros:
+   - POSIX APIs.
+   - UART driver is available in most VM, development-friendly.
```

Comment:
What do we mean by "development-friendly"?

Reply:
Fixed. POSIX can be called directly to realize vUART communication without additional development drivers